extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+
+Availability data is scoped to a subtree. This ensures that the size of each bitstream is bounded to a reasonable size.
+
+### Tile Availability
+
+**Tile availability** is a bitstream that determines which tiles exist within a subtree. There is one bit for each subtree node. A 1 indicates that a tile is available, while a 0 indicates that a tile is unavailable.
+
+![Tile Availability](figures/tile-availability.jpg)
+
+In the diagram above, colored nodes indicated available tile, while nodes with dashed outlines are unavailable tiles.
+
+If a tile is marked as available, more information about the tile, such as its content or children can be queried. If a tile is marked as unavailable, the tile must be skipped.
+
+### Content Availability
+
+**Content availability** is a bitstream that determines which tiles have an associated content file. Like tile availability, there is one bit for each subtree node. A 1 indicates a content file exists for this tile, while a 0 indicates that no content file exists.
+
+![Content Availability](figures/content-availability.jpg)
+
+The purpose of content availability is to check if a file exists before making a network request. If content is marked as unavailable, the network request for that file must be skipped.
+
+A content availability bit must only be set if the corresponding tile availability bit is set. Otherwise, it would be possible to specify content files that are not reachable by the tiles of the tileset. The content availability bitstream can be validated by checking that the following equation holds true:
+
+```
+contentAvailability & ~tileAvailability === 0
+```
+
+where `&` is the bitwise AND operation and `~` is the bitwise NOT operation.
+
+### Child Subtree Availability
+
+**Child subtree availability** is a bitstream that determines what subtrees can be reached from this subtree. There are `N` bits for every node in the bottom-most level of the subtree, where `N` is the branching factor of the tree. A 1 means there is a child subtree available at that position in the tree. Meanwhile, a 0 means there is no subtree available.
+
+![Child Subtree Availability](figures/subtree-availability.jpg)
+
+Child subtree availability is used to determine whether further subtree files exist before making network requests. If a child subtree availability bit is 0, any network request for that subtree must be skipped.
+
+## Subtree JSON Files
+
+A **subtree JSON file** describes where the availability information for a single subtree is stored.
+
+Each subtree JSON file contains the following information:
+
+* The location of a bitstream for tile availability (if not `constant`)
+* The location of a bitstream for content availability (if not `constant`)
+* The location of a bitstream for child subtree availability (if not `constant`)
+
+### Buffers and Buffer Views
+
+OUTLINE:
+- Which spec to reference? 3DTILES_binary_buffers or core metadata?
```

Comment:
@sanjeetsuhag @ErixenCruz we should discuss this today.

Reply:
This should reference 3DTILES_binary_buffers, though that [draft spec](https://github.com/CesiumGS/3d-tiles/tree/3DTILES_binary_buffers/extensions/3DTILES_binary_buffers) doesn't really explain what buffers/bufferViews are...

Reply:
No, we're just going to describe this inline. If possible we want only implicit tiling to need buffer views.